tutorial/westeros/westeros_share_constraint.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Westeros Tutorial Part 3a - Share Constraints\n",
```

Comment:
Maybe you need to update the number of tutorial to 3b, because fossil resources are now 3a.

Reply:
done